scripts/translate_docx.py

```
@@ -0,0 +1,366 @@
+                processing_time,
+                cache_stats,
+                audit_path
+            )
+
+        # Apply style fixes if available
+        if STYLE_MODULES_AVAILABLE and output_path.exists():
+            logger.info("Applying style fixes...")
+            try:
+                apply_style_fixes(str(output_path))
+                logger.info("Style fixes applied successfully to output file")
+            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)